libcontainer/devices/devices_unix.go

```diff
@@ -94,6 +94,9 @@ func getDevices(path string) ([]*configs.Device, error) {
 			if err == ErrNotADevice {
 				continue
 			}
+			if os.IsNotExist(err) {
```

Comment:
@rhvgoyal 

> I think we should put a comment that this function goes through a list of devices and if some devices are not present, it ignores them.

Ignoring is worse than returning `ErrNotADevice`. It is true that a path that doesn't exist is not a device. :P